lib/v2/datasource/remote/model/planted_model.dart

```diff
@@ -0,0 +1,21 @@
+class PlantedModel {
+  final double quantity;
```

Comment:
this model was a string before. 

It comes from the server as  `400 SEEDS`

Well, we dont need the `SEEDS` in the model. leave that up to the mapper to decide how to show the data in the UI